test/e2e/simple_ovs_bridge.go

```diff
@@ -0,0 +1,60 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Simple OVS bridge", func() {
+	Context("when desiredState is configured", func() {
```

Comment:
This context does not do anything. Could you remove it and add the description to underlying Contexts?